javascript-and-json/main.js

```diff
@@ -0,0 +1,87 @@
+// eslint-disable-next-line no-unused-expressions
```

Comment:
Don't add `eslint` rules without asking me! If you get an error while committing, it's probably for a good reason!